utils/measure/runner/light.py

```diff
@@ -130,8 +130,34 @@ def run_color_mode(
         with open(measurement_info.csv_file, file_write_mode, newline="") as csv_file:
             csv_writer = CsvWriter(csv_file, color_mode, write_header_row)
 
-            if measurement_info.is_resuming is None:
-                self.light_controller.change_light_state(ColorMode.BRIGHTNESS, on=False)
+            _LOGGER.info(
```

Comment:
Please extract this code, until `time.sleep(config.SLEEP_INITIAL)` to a separate method, something like `initialize_light_for_color_mode`.
This method `run_color_mode` is getting quite massive and makes it harder to reader.
Would be nice to split it up in more granualar functions.

Reply:
Sure, will do that later when I'm home again.

Reply:
@RubenKelevra Could you still make this change?

Reply:
Done.